groovy/postInit/mod/OpenSecurity.groovy

```diff
@@ -107,8 +98,7 @@ for (solder in soldering_alloys) {
 
 	mods.gregtech.assembler.recipeBuilder()
 			.inputs(metaitem('hull.mv'))
-			.inputs(ore('oc:cpu1'))
-			.inputs(ore('oc:ram1'))
+			.inputs(ore('circuitMv'))
 			.inputs(item('opensecurity:mag_card'))
```

Comment:
I don't think it really makes sense to have a card inside of a card reader at all times; I would prefer an emitter, although with a configuration circuit. Same for the concrete recipe.

Reply:
My latest commit addresses it.